### DISCUSSION TITLE
Nestjs How to fix @Transactional for unit test jest

### DISCUSSION THREAD

**Discussion by maxval030** [Aliheym/typeorm-transactional#49] [Q&A]
Message test code 
`No storage driver defined in your app ... please call initializeTransactionalContext() before application start.`

**Comment by phu-dangn-otsv** [comment]
Hey @maxval030 you just mock the implementation of it:
```
jest.mock('typeorm-transactional', () => ({
  Transactional: () => () => {},
}));
```
